client/extensions/woocommerce/woocommerce-services/components/text-field/index.js

```diff
@@ -48,7 +50,7 @@ TextField.propTypes = {
 	id: PropTypes.string.isRequired,
 	title: PropTypes.node,
 	description: PropTypes.string,
-	value: PropTypes.string.isRequired,
```

Comment:
`value` is not required. On uncontrolled components, it is not necessary to pass a `value` prop: https://reactjs.org/docs/uncontrolled-components.html